packages/mdc-snackbar/package.json

```diff
@@ -15,5 +15,5 @@
     "@material/rtl": "^0.1.1",
     "@material/theme": "^0.1.1",
     "@material/typography": "^0.1.1"
-  }
+  } 
```

Comment:
can we remove this? this way a whitespace change in a different component doesn't bleed into this PR